thorlcr/master/thgraphmanager.cpp

```diff
@@ -921,13 +922,17 @@ bool CJobManager::executeGraph(IConstWorkUnit &workunit, const char *graphName,
         updateWorkunitStat(wu, SSTgraph, graphName, StTimeElapsed, graphTimeStr, graphTimeNs, wfid);
 
         addTimeStamp(wu, SSTgraph, graphName, StWhenFinished, wfid);
-        
+
+        fatalHdlr.setown(job->clearFatalHandler());
+
```

Comment:
It only need to clear it before the job is destroyed, so doesn't need to do this here.